datadog_checks_base/datadog_checks/base/utils/agent/utils.py

```diff
@@ -0,0 +1,17 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+
+def should_profile_memory(datadog_agent, check_name):
+    tracemalloc_whitelist = datadog_agent.get_config('tracemalloc_whitelist') or ''
+    if tracemalloc_whitelist:
+        tracemalloc_whitelist = [check for check in tracemalloc_whitelist.split(',') if check]
+
+    tracemalloc_blacklist = datadog_agent.get_config('tracemalloc_blacklist') or ''
+    if tracemalloc_blacklist:
+        tracemalloc_blacklist = [check for check in tracemalloc_blacklist.split(',') if check]
```

Comment:
The `check` after the if might need to to be striped too.

Unfortunately seems not possible to avoid stripping twice with list comp. So you might need to:

```
        tracemalloc_blacklist = []
        for check in tracemalloc_blacklist.split(','):
            check = check.strip()
            if check:
                tracemalloc_blacklist.append(check)
```

or 

```
        tracemalloc_blacklist = [check.strip() for check in tracemalloc_blacklist.split(',')]
        tracemalloc_blacklist = [check for check in tracemalloc_blacklist if check]
```